website/views/shared/tasks/index.jade

```diff
@@ -6,7 +6,7 @@ include ./task_view/mixins
 script(id='templates/habitrpg-tasks.html', type="text/ng-template")
   .tasks-lists.container-fluid
     .row
-      .col-md-3.col-sm-6(ng-repeat='list in lists', ng-class='::{"rewards-module": list.type==="reward"}')
+      .col-md-3.col-sm-6(ng-repeat='list in lists track by $index', ng-class='::{"rewards-module": list.type==="reward"}')
```

Comment:
Until the task migration is complete, we cannot use `track by $index` because some tasks have duplicate ids and Angular breaks if that happens while `track by` is turned on.